server/controllers/finance/debtors/groups/index.js

```diff
@@ -314,9 +314,8 @@ function loadInvoices(params) {
     ) AS i
     JOIN project ON i.project_id = project.id
     JOIN entity_map ON i.entity_uuid = entity_map.uuid
-    WHERE i.uuid NOT IN (
```

Comment:
This query does need to be optimized!

Can we just do
```sql
WHERE i.reversed <> 1;
```
?

Reply:
👍 Much better 

NOT IN : 50seconds 
LEFT JOIN : 872ms
i.reversed <> 1 : 473ms